staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/internal/generic/controller.go

```diff
@@ -188,7 +194,11 @@ func (c *controller[T]) Run(ctx context.Context) error {
 }
 
 func (c *controller[T]) HasSynced() bool {
-	return c.informer.HasSynced()
+	hasSynced := c.hasSynced.Load()
+	if hasSynced == nil {
+		return false
+	}
+	return hasSynced.(func() bool)()
```

Comment:
this is an example of where the handler is queueing objects for async processing we'd need to track actually completed handling the initial set, right?

Reply:
Yes, I missed this -- fixed in last commit